components/CommunityCard/index.js

```diff
@@ -41,12 +42,18 @@ const CommunityCard = ({ canModify, community, credentials }) => {
   }
 
   const publishCommunity = async () => {
-    setHeader(credentials)
-    await api.put(`/community/publish/${_id}`, {
-      status: 'published'
-    })
-    sendNotification('publish')
-    Router.push('/')
+    try {
+      setHeader(credentials)
+      await api.put(`/community/publish/${_id}`, {
+        status: 'published'
+      })
+      sendNotification('publish')
+      Router.push('/')
+    } catch (e) {
+      toast.warn('Sessão expirada. Você será redirecionado para página de login!')
```

Comment:
@theleoad só um detalhe que esqueci de falar contigo anteriormente. Precisamos, dentro do catch, verificar se o erro foi este mesmo 401. Se não for não precisamos deslogar, mas apresentar uma mensagem genérica: `Ops! Ocorreu um erro ao publicar uma comunidade, por favor, tente novamente.`